client/index.js

```diff
@@ -1,41 +1,39 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import React from 'react';
+import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import { Router, browserHistory } from 'react-router';
+
+import { BrowserRouter as Router } from 'react-router-dom';
 import { PersistGate } from 'redux-persist/es/integration/react';
-import configureStore from './store/configureStore';
-import routes from './routes';
-import axios from 'axios';
 import { ApolloProvider } from 'react-apollo';
+import axios from 'axios';
+
+import configureStore from './store/configureStore';
+import Routes from './routes';
+import Client from './client';
 
 // styles
 import './assets/style.scss';
-import Client from "./client";
 
 // store
 const { persistor, store } = configureStore();
 
-if (localStorage.getItem('a_socials')) {
-  const token = JSON.parse(localStorage.getItem('a_socials'));
-  const jwt = token.jwt;
-  axios.defaults.headers.common['Authorization'] = `JWT ${jwt}`;
+if (window.localStorage.getItem('a_socials')) {
```

Comment:
since we are building for the browser you don't have to use `window` explicitly, doing this in `.eslintrc` would solve the eslint error: 
```
{
  ...
  "env": {
     ...
     "browser": true,
     ...
  }
  ...
}
```
```